aws-predictions/src/androidTest/java/com/amplifyframework/predictions/aws/AWSPredictionsInterpretTest.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.aws;
+
+import com.amplifyframework.predictions.PredictionsException;
+import com.amplifyframework.predictions.models.Entity;
+import com.amplifyframework.predictions.models.KeyPhrase;
+import com.amplifyframework.predictions.models.Language;
+import com.amplifyframework.predictions.models.LanguageType;
+import com.amplifyframework.predictions.models.Sentiment;
+import com.amplifyframework.predictions.models.SentimentType;
+import com.amplifyframework.predictions.models.SpeechType;
+import com.amplifyframework.predictions.models.Syntax;
+import com.amplifyframework.predictions.options.InterpretOptions;
+import com.amplifyframework.predictions.result.InterpretResult;
+import com.amplifyframework.testutils.Assets;
+import com.amplifyframework.testutils.sync.SynchronousPredictions;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Tests that Predictions interpret delivers a non-null result
+ * for valid input.
+ */
+public final class AWSPredictionsInterpretTest {
+
+    private static SynchronousPredictions predictions;
+
+    /**
+     * Configure Amplify and Predictions plugin before the tests.
+     * @throws Exception if configuration fails
+     */
+    @BeforeClass
+    public static void setUpOnce() throws Exception {
```

Comment:
Note: if you get this to a fresh `PredictionsCategory` each time, then you can make it as `@Before public void setup()`, which will help guarantee fresh state for each individual test.

(Once we get rid of the process singletons, we should kill all of the `@BeforeClass`. Yea, they might be a little more performance than doing setup before each individual test, but at the cost of potential cross-pollution of the individual tests.)